### ISSUE TITLE
Crashes on android 7.0 and below due to lack of List.sort Api

### ISSUE THREAD

**Issue by choiman1559** [srikanth-lingala/zip4j#250]
Hello, Thanks for your work!

```List.sort``` APIs were added to Android with API 26 (8.0, Oreo). Thus, latest version of zip4j will work only on Android 8.0 and up for Android devices. Is there any solution for it? I need Android 7 or lower support to develop and maintain my app, but this problem makes it impossible.

Originally posted by @choiman1559 in https://github.com/srikanth-lingala/zip4j/issues/4#issuecomment-707142122

```
Caused by java.lang.NoSuchMethodError: No interface method sort(Ljava/util/Comparator;)V in class Ljava/util/List; or its super classes (declaration of 'java.util.List' appears in /system/framework/core-libart.jar)
       at net.lingala.zip4j.tasks.AbstractModifyFileTask.cloneAndSortFileHeadersByOffset(AbstractModifyFileTask.java:76)
       at net.lingala.zip4j.tasks.RemoveFilesFromZipTask.executeTask(RemoveFilesFromZipTask.java:52)
       at net.lingala.zip4j.tasks.RemoveFilesFromZipTask.executeTask(RemoveFilesFromZipTask.java:21)
       at net.lingala.zip4j.tasks.AsyncZipTask.performTaskWithErrorHandling(AsyncZipTask.java:46)
       at net.lingala.zip4j.tasks.AsyncZipTask.execute(AsyncZipTask.java:40)
       at net.lingala.zip4j.tasks.AbstractAddFileToZipTask.removeFile(AbstractAddFileToZipTask.java:266)
       at net.lingala.zip4j.tasks.AbstractAddFileToZipTask.removeFilesIfExists(AbstractAddFileToZipTask.java:251)
       at net.lingala.zip4j.tasks.AbstractAddFileToZipTask.addFilesToZip(AbstractAddFileToZipTask.java:64)
       at net.lingala.zip4j.tasks.AddFolderToZipTask.executeTask(AddFolderToZipTask.java:28)
       at net.lingala.zip4j.tasks.AddFolderToZipTask.executeTask(AddFolderToZipTask.java:17)
       at net.lingala.zip4j.tasks.AsyncZipTask.performTaskWithErrorHandling(AsyncZipTask.java:46)
       at net.lingala.zip4j.tasks.AsyncZipTask.execute(AsyncZipTask.java:40)
       at net.lingala.zip4j.ZipFile.addFolder(ZipFile.java:364)
       at net.lingala.zip4j.ZipFile.addFolder(ZipFile.java:339)
       at com.fqxd.gftools.features.decom.PatchTask.doInBackground(PatchTask.java:130)
       at com.fqxd.gftools.implement.AsyncTask$3.call(AsyncTask.java:223)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at com.fqxd.gftools.implement.AsyncTask$SerialExecutor.lambda$execute$0$AsyncTask$SerialExecutor(AsyncTask.java:136)
       at com.fqxd.gftools.implement.-$$Lambda$AsyncTask$SerialExecutor$FGB3Rik2507exmLqsfc4mPQNB68.run(lambda)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:818)
       ```

**Comment by srikanth-lingala** [comment]
looking into it

**Comment by srikanth-lingala** [comment]
Fixed in v2.6.4 released today
